docs/user/custom-domains.rst

```diff
@@ -4,123 +4,22 @@ Custom Domains
 Custom domains allow you to serve your documentation from your own domain.
 This is great for maintaining a consistent brand for your documentation and application.
 
-By default, your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>` using the project's :term:`slug`:
+By default, your documentation is served from a Read the Docs :ref:`subdomain <hosting:Subdomain support>` using the project's :term:`slug`:
 
 * ``<slug>.readthedocs.io`` for |org_brand|
 * ``<slug>.readthedocs-hosted.com`` for |com_brand|.
 
 For example if you import your project and it gets the :term:`slug` ``example-docs``, it will be served from ``https://example-docs.readthedocs.io``.
 
-.. contents:: Contents
-    :local:
+How does it work?
+-----------------
 
-Adding a custom domain
-----------------------
+If you own a domain and would like to use it for your documentation, Read the Docs requires only :ref:`a single edit <adding_domain>` to the DNS settings of the domain.
+After this, your documentation will be served from your domain and SSL will also be configured automatically.
 
-To setup your custom domain, follow these steps:
+In case you change your domain name, your documentation can have multiple secondary domains but only one canonical domain name.
+Additional domains or subdomains will redirect to the canonical domain.
```

Comment:
This section is new.

Reply:
This section is a great addition. This is content that we've been missing for a while, and we don't always explain to the user (in terms the user is familiar with) what we actually do.

Reply:
Great! I'm glad that we could do the split and only have to write 1 new subsection, even though there's a lot of review feedback.

I hope I can present some PRs that invite a faster process after this.